edgelet/Cargo.lock

```diff
@@ -22,6 +22,15 @@ dependencies = [
  "winapi 0.3.5 (registry+https://github.com/rust-lang/crates.io-index)",
 ]
 
+[[package]]
```

Comment:
Do you need new entries in THIRDPARTYNOTICES ?

Reply:
Yes, I do - I have more to add, so I included this as a task on my PBI.